common/config/rush/variants/stable/common-versions.json

```diff
@@ -26,8 +26,9 @@
     // "some-library": "1.2.3"
     // This is the version for stable build (please also update allowedAlternativeVersions below)
     "@azure/communication-calling": "^1.16.3",
-    "@azure/communication-common": "2.2.1",
-    "@azure/communication-chat": "^1.3.2"
+    "@azure/communication-common": "2.3.0",
```

Comment:
remove also

Reply:
I would like to leave the plumbing in just in case the Calling SDK make s the choice to go back to the beta common package